app/logic/Mail/OWA/OWAFolder.ts

```diff
@@ -1,9 +1,461 @@
-import { Folder } from "../Folder";
+import { Folder, SpecialFolder } from "../Folder";
 import { OWAEMail } from "./OWAEMail";
+import type { OWAAccount } from "./OWAAccount";
+import { SQLFolder } from "../SQL/SQLFolder";
+import { base64ToArrayBuffer, assert } from "../../util/util";
+import { sanitize } from "../../../../lib/util/sanitizeDatatypes";
+import { ArrayColl, Collection } from "svelte-collections";
+
+export const kMaxCount = 50;
 
 export class OWAFolder extends Folder {
+  account: OWAAccount;
 
   newEMail(): OWAEMail {
     return new OWAEMail(this);
   }
+
+  fromJSON(json: any) {
```

Comment:
```suggestion
  fromOWA(json: any) {
```

Reply:
I didn't see you complain about `fromXML` in the EWS code...